scripts/Build-DockerContainer.ps1

```diff
@@ -0,0 +1,92 @@
+param (
```

Comment:
It uses the REST API to dispatch the Docker build and update the container in the cloud, with appropriate values extracted from the app's code. It would be equivalent to a set of grep commands to read values from the file system followed by a curl command that dispatches the build.

This script isn't strictly necessary. It's just a utility for a developer (me!) who needs to update and test from a specific branch. It's possible to dispatch the docker build from the GitHub workflows menu (on GitHub) instead. But this does a few checks for consistency, so for me it's reassuring to have.

I will add a description.